canonical-web-and-design/ubuntu.com
# Update to use inclusive language and add github action

## Done

- Updates language to only use inclusive language. Based on the [inclusive language list](https://docs.google.com/document/d/1mJUa1VJHOMWa723dmKmNjSKGW-nlBv9xktzGZZwacVo/edit#). Kubernetes has been excluded, as have links.
- Adds the github action to run on each PR.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check there are no occurrences of the worst cases of non-inclusive language, these can be found on the [inclusive language overview](https://docs.google.com/document/d/1beNWplggAw-FWgvsjd3IxLh3t28TpBWgnkLoq5kYDBU/edit#) 



## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4407
